build.gradle

```diff
@@ -1,12 +1,14 @@
 buildscript {
-    ext.kotlin_version = '1.3.20'
-    ext.ktor_version = '1.1.2'
+    ext.kotlin_version = '1.3.61'
+    ext.ktor_version = '1.2.6'
+    ext.serialization_version = '0.14.0'
     repositories {
         jcenter()
         maven { url 'https://dl.bintray.com/jetbrains/kotlin-native-dependencies' }
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
```

Comment:
how come we're using the `kotlin_version` here when we have `serialization_version`?

Reply:
that's what jetbrains have in manual:
https://github.com/Kotlin/kotlinx.serialization#gradle

and then for `kotlinx-serialization-runtime`
https://github.com/Kotlin/kotlinx.serialization#mavenjvm